sources/packages/backend/libs/utilities/src/logger/logger.service.ts

```diff
@@ -0,0 +1,7 @@
+import { Injectable, ConsoleLogger, Scope } from "@nestjs/common";
+
+/**
+ * Common log across entire solution.
+ */
+@Injectable({ scope: Scope.TRANSIENT })
```

Comment:
for my info what is `scope: Scope.TRANSIENT`

Reply:
From Nestjs docs: "Transient providers are not shared across consumers. Each consumer that injects a transient provider will receive a new, dedicated instance."
Basically, a new instance is created every time that it is injected.